File: Python/1043.py
num = input().split()

A = float(num[0])
B = float(num[1])
C = float(num[2])
   
if (A>=B +C or B>= A+C or C>=B+A):
    print(f'Area = {((A+B)*C)/2:.1f}')
else: 
    print(f'Perimetro = {A+B+C:.1f}')
   

 

File: C++/1043.cpp
#include <iostream>
 #include <iomanip>
using namespace std;
 
int main() {
 
   double A, B , C;
   cin >> A >> B>> C;
   
   if (A>=B +C || B>= A+C || C>=B+A){
 
  cout << fixed << setprecision(01) << "Area = " << ((A+B)*C)/2 << endl; 
   }
   
   
   else  { 
        cout <<fixed << setprecision(01) << "Perimetro = " << A+B+C << endl;
   }

 
    return 0;
}